src/main/java/com/minecolonies/coremod/client/gui/modules/WindowListRecipes.java

```diff
@@ -196,11 +196,17 @@ public void updateElement(final int index, @NotNull final Pane rowPane)
                 }
 
                 final Text intermediate = rowPane.findPaneOfTypeByID("intermediate", Text.class);
-                if(recipe.getIntermediate() != Blocks.AIR)
+                intermediate.setVisible(false);
+                if(recipe.getRequiredTool() != ToolType.NONE)
+                {
+                    intermediate.setText(recipe.getRequiredTool().getDisplayName());
+                    intermediate.setVisible(true);
+                }
+                else if(recipe.getIntermediate() != Blocks.AIR)
                 {
                     intermediate.setText(recipe.getIntermediate().getName());
+                    //intermediate.setVisible(true);
```

Comment:
uncommented?

Reply:
I left it in but commented because the previous code here would never have set it to true.  I assumed this was intentional.

Reply:
was probably more of a derp tbh. should be true here and then false in the else =D

Reply:
I also figured that it was kinda unnecessary now that _most_ of the smelting recipes are on a separate tab (though there are a very few exceptions).  Since all this would have done is shown "Furnace" or blank.